### DISCUSSION TITLE
How can I migrate workspace.json from v1 to v2?

### DISCUSSION THREAD

**Discussion by LuckeeDev** [nrwl/nx#7150] [General]
I have an old project which I created when `workspace.json` version was still v1. How can I convert this file to the new `workspace.json` format? What has changed exactly?

**Comment by brandonroberts** [comment]
Running `nx format` after migrating to the latest version with convert the file for you. The main changes are:

- `architect` is changed to `targets`
- `schematics` is changed to `generators`
- `builders` is changed to `executors`

**Comment by brandonroberts** [comment]
You also get the option to break down your workspace.json into `project.json` files under each app/lib

**Comment by LuckeeDev** [comment]
`nx format` did nothing for me, but I manually managed to move to v2, thank you very much!

**Comment by LuckeeDev** [comment]
Sorry to bother, but I now get two warnings:
```
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
```
and
```
Warning: C:/src/projects/web/csl/node_modules/@editorjs/editorjs/types/configs/conversion-config.ts is part of the TypeS
cript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.
```

Any idea how to solve this? (For the first one, running `npx browserslist@latest --update-db` doesn't remove the warning.
